chapter_natural-language-processing-pretraining/subword-embedding_vn.md

```diff
@@ -40,7 +49,11 @@ In view of this, fastText :cite:`Bojanowski.Grave.Joulin.ea.2017` proposes the m
 thereby attempting to introduce morphological information in the skip-gram model in word2vec.
 -->
 
-*dịch đoạn phía trên*
+Trong word2vec, ta không trực tiếp sử dụng thông tin hình thái học.
+Trong cả mô hình skip-gram và bag-of-word liên tục, ta sử dụng các vector khác nhau để biểu diễn các từ ở các dạng khác nhau.
```

Comment:
```suggestion
Trong cả mô hình skip-gram và túi từ liên tục, ta sử dụng các vector khác nhau để biểu diễn các từ ở các dạng khác nhau.
```

Reply:
Mình nghĩ bag-of-word nên giữ nguyên như từ skip-gram vì nó đã trở thành một thuật ngữ chuyên dụng hay dùng.

Reply:
em thấy trong glossary dịch vậy, nếu anh thấy ko ổn có thể mở discussion thảo luận lại.